src/utils/changeDayFormat.js

```diff
@@ -0,0 +1,22 @@
+const changeDayFormat = (dateNumber) => {
+  switch (dateNumber) {
+    case 0:
+      return "일요일";
+    case 1:
+      return "월요일";
+    case 2:
+      return "화요일";
+    case 3:
+      return "수요일";
+    case 4:
+      return "목요일";
+    case 5:
+      return "금요일";
+    case 6:
+      return "토요일";
+    default:
+      throw Error("유효하지 않은 값입니다.");
```

Comment:
```suggestion
      throw Error("유효하지 않은 요일 정보입니다.");
```
메시지에 요일 관련 정보임을 포함시키면 어떨까요? 에러 발생시 상황 파악이 더 수월할 것 같습니다. 
에러가 대략 위치를 알려주기는 하지만, 다른 코드에도 쓰일 수 있는 일반적인 메시지 보다 맥락이 담긴 에러 메시지가 더 좋은 기준이 될 것 같다고 생각했어요. 관련해서 참고자료를 더 찾아보고 싶었는데, 아직 발견하지 못 했어요. 의견 나눠보면 좋겠습니다 :)

Reply:
저도 요일로 명확하게 알려주는 게 더 좋은 방법인 것 같습니다.

Reply:
수정 완료 하였습니다 :)